### ISSUE TITLE
Is there a way to create a "password field"?

### ISSUE THREAD

**Issue by tman540** [npcole/npyscreen#84]
I would like to create a password field. For example, when typing, the text is replaced with asterisks (*) or text is not shown (like Linux terminals). Is there a way to do this?

**Comment by thorstenkampe** [comment]
Yes, please read the documentation: https://npyscreen.readthedocs.io/widgets-text.html?highlight=titlepassword
